src/components/Toast/__snapshots__/Toast.test.tsx.snap

```diff
@@ -0,0 +1,46 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Toast /> Alert story renders snapshot 1`] = `
```

Comment:
Weird that the button's not showing up in here 🤔

Reply:
That's true, but the v0 test file also didn't have buttons showing up 😓 , think it can be addressed in a separate ticket

Reply:
Interestingly, buttons also not appearing in chromatic

Reply:
Oh yeah I see it in chromatic https://www.chromatic.com/test?appId=61313967cde49b003ae2a860&id=624b4fc67a4d27003afb577d